### PR TITLE
Default database config added in the config file

### PR DESCRIPTION
**Pull request by rajul**

### CODE REVIEW

**Comment by ehsteve** [comment]
:+1:

File: sunpy/util/config.py [hunk 1]
```diff
@@ -25,6 +25,11 @@ def load_config():
     if not config.has_option('general', 'working_dir'):
         config.set('general', 'working_dir', os.path.join(_get_home(), "sunpy"))
 
+    # Specify the database url as a default so that the user's home
+    # directory can be located in an OS-independent manner
+    if not config.has_option('database', 'url'):
+        config.set('database', 'url', "sqlite:///" + os.path.join(_get_home(), "sunpy/sunpydb.sqlite"))
```
Comment (dpshelio) [inline sunpy/util/config.py]: Does this works in Windows?

Comment (Cadair) [inline sunpy/util/config.py]: nope

Comment (ayshih) [inline sunpy/util/config.py]: The string looks funny, but Python still handles it correctly.

**Pull request merged**